### PR TITLE
add enrich docs

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: search/enrich/enrich.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Enrich
+
+The `enrich` module enables adding enumerated values to all entries in a pipeline within the query or using resources. `enrich` can be used to annotate data with constant enumerated values, such as "User=admin", in order to simplify visualization and reporting, pivoting within compound queries, and working with nontemporal data.
```
Comment (floren) [inline search/enrich/enrich.md:3]: "enables adding" is clunky. what about "The `enrich` module can add enumerated values to each entry in a pipeline; these values can be specified in the module arguments or can come from a resource."

File: search/enrich/enrich.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The `enrich` module can extract columns from a CSV or lookup table resource. When using resources, only the first row will be used by `enrich`. Columns can be specified, and all columns will be used if none are specified. If a column conflicts with an existing enumerated value, it will only be overwritten if using the `-o` flag. 
+
+For example, to use columns "foo" and "bar" from a resource "data":
+
+```
+tag=jsondata val | enrich -r data foo bar | table val foo bar"
+```
+
+NOTE: Only the first row of resources are used with the `enrich` module.
```
Comment (floren) [inline search/enrich/enrich.md:34]: Add something like "When using CSV files, note that the first row refers to the first row of actual data, not the row containing column names"?

**Review by floren — COMMENTED** [review]
Document the "as" option too?

File: search/enrich/enrich.md (hunk 1 above)
Comment (djfritz) [reply search/enrich/enrich.md]: done

File: search/enrich/enrich.md (hunk 2 above)
Comment (djfritz) [reply search/enrich/enrich.md]: added "(excluding column headers when using CSV resources)"

**Comment by djfritz** [comment]
@floren PTAL

**Review by floren — APPROVED** [review]

**Pull request merged**